srv/salt/ceph/apparmor/default-enfroce.sls

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
this file should be named `default-enforce.sls` probably? (typo in the filename)